include/swift/Runtime/Metadata.h

```diff
@@ -660,7 +660,7 @@ const ExtraInhabitantsValueWitnessTable METATYPE_VALUE_WITNESS_SYM(Bo); // Built
 
 /// Return the value witnesses for unmanaged pointers.
 static inline const ValueWitnessTable &getUnmanagedPointerValueWitnesses() {
-#ifdef __LP64__
+#ifdef __POINTER_WIDTH__ == 64
```

Comment:
Oops, I uploaded the wrong version  This should be `#if __POINTER_WIDTH__ == 64`.